### ISSUE TITLE
Thumbnails for Log In Required items break after logged in session exceeds 24 hours

### ISSUE THREAD

**Issue by nikdragovic** [emory-libraries/dlp-lux#625]
Thumbnails break after a given period of time, in all environments. This is possibly due to the expiration of a cookie generated during authentication that enables access to these thumbnail images.

See #561 for prior work and additional details.
Depending on outcomes of this ticket, related ticket #618 will be re-evaluated.

## Acceptance Criteria
- [ ] Investigation produces a possible cause and/or solution. Surface documentation about authentication cookie as appropriate.
- [ ] Thumbnails should not break before a session expires.

**Comment by nikdragovic** [comment]
As of 8am 7/20, when I log in in an incognito window, I am not able to load Log In Required thumbnails in test. Prod and arch seem to be working normally.

![Screen Shot 2020-07-20 at 7.58.43 AM.png](https://images.zenhubusercontent.com/5ba135cf945aef49576d50f2/702dcea9-9ac6-4b3c-b2cd-739a3c6325eb)

**Comment by nikdragovic** [comment]
I successfully rendered this error message when I kept an older session logged in. I also logged out of and back into the incognito session and it appears to be working properly. I will keep checking to make sure there is no unexpected behavior.
 
![Screen Shot 2020-07-20 at 8.14.05 AM.png](https://images.zenhubusercontent.com/5ba135cf945aef49576d50f2/49dfcdd6-9a61-4ced-a8ad-7aa61da54179)
![Screen Shot 2020-07-20 at 8.14.41 AM.png](https://images.zenhubusercontent.com/5ba135cf945aef49576d50f2/d6c0000d-0bb3-4147-8e4d-3fce4795a7a1)
